### ISSUE TITLE
Add Stopwatch

### ISSUE THREAD

**Issue by amishra026** [subodhk01/a2oj#7]
I was wondering if we could have a Stopwatch that starts as soon as the user clicks on the problem. This will help the user improve his/her timing which might matter to him/her. The time required will only be stored for a particular session, and not forever (so no username/password required). To end the timer, the time of submission on CF can be used.

**Comment by subodhk01** [comment]
Yeah this can be helpful for a great number of users, Do you want to try and implement this feature?

**Comment by amishra026** [comment]
I'd like to try, although it would be my first contribution to open source. How should I start?

**Comment by subodhk01** [comment]
Start by forking the repository, then clone the forked repo onto your local machine. Then setup a python environment to install all the dependencies( refer https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#using-requirements-files ), all the dependencies are in `requirements.txt` file. Then simply run `python manage.py runserver` to start a local server and access it via `localhost:8000`
